editor/plugins/script_text_editor.cpp

```diff
@@ -1339,6 +1339,17 @@ void ScriptTextEditor::_edit_option(int p_op) {
 	}
 }
 
+Vector2i ScriptTextEditor::get_cursor_position() {
```

Comment:
Rather then adding a new method, it would be beneficial to use `get_edit_state()` instead, this way it's agnostic to the current editor. The state has `state["row"]` and `state["column"]` for the caret position.

Reply:
Interesting. Didn't know about that.

I'll take a look asap.